CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 1.0.0
+
+Security and privacy improvements:
+- Faster and more secure 2-layer E2E encryption, using double ratchet algorithm with Curve448 and Curve25519 for key agreement.
+- Encryption of the traffic from server to recipient to avoid cipher-text correlation between sent and received.messages.
+- TLS 1.2+ with tls-unique binding to avoid the possibility of command replay.
```

Comment:
I think it should be just TLS 1.2 without plus for now
```suggestion
- TLS 1.2 with tls-unique binding to avoid the possibility of command replay.
```

Reply:
it'll be fixed soon